src/Build/Resources/Strings.resx

```diff
@@ -331,6 +331,18 @@
   </data>
   <data name="TryingExtensionsPath" xml:space="preserve">
     <value>Trying to import {0} using extensions path {1}</value>
+  </data>
+    <data name="OverrideUsingTaskElementCreated" xml:space="preserve">
+    <value>MSB4275: Created an override using task: {0}</value>
+    <comment>{StrBegin="MSB4275: "}</comment>
+  </data>
+  <data name="DuplicateOverrideUsingTaskElement" xml:space="preserve">
+    <value>MSB4276: Attempted to create multiple overrides of the same task: {0}</value>
```

Comment:
Is there a test that this gets thrown? I didn't see one.

Reply:
Just added it :)